pkgs/by-name/ma/mailsy/package.nix

```
@@ -0,0 +1,32 @@
+{
+  buildNpmPackage,
+  fetchFromGitHub,
+  lib,
+}:
+buildNpmPackage rec {
+  pname = "mailsy";
+  version = "5.0.0";
+
+  src = fetchFromGitHub {
+    owner = "BalliAsghar";
+    repo = "Mailsy";
+    rev = version;
+    hash = "sha256-RnOWvu023SOcN83xEEkYFwgDasOmkMwSzJ/QYjvTBDo=";
+  };
+
+  npmDepsHash = "sha256-ljmqNmLvRHPdsKyOdDfECBXHTIExM6nPZF45lqV+pDM=";
+
+  npmFlags = ["--legacy-peer-deps"];
+
+  dontNpmBuild = true;
+
+  patches = [./fix-file-lookup.patch];
```

The patch is pretty simple, so we can just do this instead:

```suggestion
  postPatch = ''
    substituteInPlace utils/index.js \
      --replace-fail 'dirname, "../data/account.json"' 'process.cwd(), "account.json"' \
      --replace-fail 'dirname, "../data/email.html"' 'process.cwd() + "email.html"'
  '';
```